stripe/_stripe_service.py

```diff
@@ -0,0 +1,13 @@
+from stripe._api_requestor import APIRequestor
+from stripe._client_options import _ClientOptions
+
+from typing import Optional
+
+
+class StripeService(object):
+    _requestor: APIRequestor
+    _options: Optional[_ClientOptions]
```

Comment:
`_options` is used by `OAuthService` to access the `client_id` client option. I suppose we could just define `_options` or `client_id` specifically on `OAuthService`, but I figured I'd keep `_options` on the service in case more services will need access to client options. Do you have a preference?

Reply:
Removed in https://github.com/stripe/stripe-python/pull/1200/commits/e5b202715ac8b173bd686d7b32d83f2b9e67808e